docs/docs/prototype-an-assistant.mdx

```diff
@@ -150,18 +196,17 @@ rules:
    - action: utter_subscribed
 ```
 
-[Learn more about rules and how to write them.](./rules.mdx)
-
+</AssistantBuilder.Code>
+</AssistantBuilder.Section>
+</AssistantBuilder.Container>
 
 ## Train and talk to your assistant
 
 Once you have reviewed the steps above, you’re ready to train your assistant. The training process generates a new machine learning model based on the training data you’ve provided.
 
 To train the assistant on the NLU data, stories, forms, rules and responses above, click the `Train` button:
 
-<TrainButton />
-
-<div id="rasa-chat-block"></div>
+<div id="rasa-chat-block" class="mdx-box-max"></div>
```

Comment:
HTML in MDX still should be in JSX format, i.e. using `className` instead of `class`

Reply:
done!